### PR TITLE
Add the possibility to create a team that contains all members of your organization

### PR DESCRIPTION
**Pull request by soerenmartius**

### CODE REVIEW

File: examples/organization/variables.tf [hunk 1]
```diff
@@ -35,6 +41,12 @@ variable "blocked_users" {
   ]
 }
 
+variable "create_all_members_team" {
+  description = "Whether or not to create a team that contains all users of the organization. The teams name can be set through the all_members_team_name"
+  type        = bool
+  default     = true
+}
+
```
Comment (mariux) [inline examples/organization/variables.tf:49]: why do we need a second trigger? wouldn't it be enough to just create the team if a team name is set?

File: main.tf [hunk 2]
```diff
@@ -37,3 +38,19 @@ resource "github_organization_project" "project" {
   name = each.value.name
   body = each.value.body
 }
+
+resource "github_team" "all" {
+  count = var.create_all_members_team ? 1 : 0
+
+  name        = var.all_members_team_name
+  description = "This teams contains all members of our organization."
+  privacy     = "closed"
```
Comment (mariux) [inline main.tf:47]: should be dynamic, but can be added in a later version

File: main.tf [hunk 3]
```diff
@@ -37,3 +38,19 @@ resource "github_organization_project" "project" {
...
+}
+
+resource "github_team_membership" "all" {
+  for_each = var.create_all_members_team ? local.memberships : {}
+
+  team_id  = github_team.all[0].id
+  username = each.key
+  role     = "member"
```
Comment (mariux) [inline main.tf:55]: should be dynamic, but can be added in a later version

File: examples/organization/variables.tf [hunk 4]
```diff
@@ -35,6 +41,12 @@ variable "blocked_users" {
   ]
 }
 
+variable "create_all_members_team" {
+  description = "Whether or not to create a team that contains all users of the organization. The teams name can be set through the all_members_team_name"
+  type        = bool
+  default     = true
```
Comment (mariux) [inline examples/organization/variables.tf:47]: this should trigger version increase to `0.2.x` as it is a new default behavior...

Comment (soerenmartius) [reply examples/organization/variables.tf:47]: but this is only an example ?

File: examples/organization/variables.tf (hunk 1 above)
Comment (soerenmartius) [reply examples/organization/variables.tf]: resolved in 3b1d9e1

File: main.tf (hunk 3 above)
Comment (soerenmartius) [reply main.tf:55]: resolved in bbfe8d0

File: main.tf (hunk 2 above)
Comment (soerenmartius) [reply main.tf]: resolved in bbfe8d0

**Comment by soerenmartius** [comment]
please review again @mariux

**Pull request merged**